generic/piranha-tree-sitter/src/test-resources/java/input/XPFlagCleanerInterfaceMethod.java

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.piranha;
+
```

Comment:
Similar to testing directory, let us create a `demo` or `examples` directory. Have multiple subdirectories based on language. Say `java`. And within it `src`, `config`, etc. Add simple source and configurations in there and script to invoke piranha that will help explain the end to end process. Then, if the README is updated to point to this demo, it will make this user friendly.